### ISSUE TITLE
[Suggestion] More informative logic functions. Tiny LCD panels/decals for text?

### ISSUE THREAD

**Issue by Kaiser-Indrasil** [tsunamayo/Starship-EVO#2129]
Since there are so many ways the game lets us use logic, I feel there is a need to better inform the player at a glance, what every button or switch does.
![Starship_EVO_NHbsJGjdmX](https://user-images.githubusercontent.com/63756057/82690173-9f79ce00-9c53-11ea-8e37-d3cfaf7bc401.png)
Instead of a simple "[f] activate", there could be a short instruction inside the switch's menu (click [right mouse button] to access it, for example) which player could set to, for instance, "Entry Ramp". That way, players could know at a first glance what every button or switch does.

Other idea would be to implement small (0.25x0.25) LCD panels or decals which would let you type a few words ("Entry Ramp" string on a 0.25x0.25 area would be still pretty legible) describing the neighbouring button's/switch's function.

**Comment by tsunamayo** [comment]
Yes I want to add a name input for each configurable get for example. Will add that soon. Also I will revamp the screen data link system.
 Thanks!
